### PR TITLE
Concat custom code fragment to config.cli

### PR DESCRIPTION
**Pull request by danifr**
Closes https://github.com/treydock/puppet-module-keycloak/issues/141

### CODE REVIEW

**Comment by danifr** [comment]
Attempt to fix the tests:

```diff
diff --git a/spec/classes/init_spec.rb b/spec/classes/init_spec.rb
index e296b9a..d5af6a7 100644
--- a/spec/classes/init_spec.rb
+++ b/spec/classes/init_spec.rb
@@ -119,17 +119,24 @@ describe 'keycloak' do
         end
 
         it do
-          is_expected.to contain_file("/opt/keycloak-#{version}/config.cli").only_with(
-            ensure: 'file',
+          is_expected.to contain_concat("/opt/keycloak-#{version}/config.cli").with(
             owner: 'keycloak',
             group: 'keycloak',
             mode: '0600',
-            content: %r{.*},
             notify: 'Exec[jboss-cli.sh --file=config.cli]',
             show_diff: 'false',
           )
         end
 
+        it do
+          is_expected.to contain_concat__fragment("config.cli-keycloak").with(
+            target: "/opt/keycloak-#{version}/config.cli",
+            content: %r{.*},
+            order: '00',
+          )
+        end
+
```

**Comment by treydock** [comment]
Those test changes look like they should work.  The changes to code look good.

File: spec/classes/init_spec.rb [hunk 1]
```diff
@@ -119,17 +119,24 @@
...
             group: 'keycloak',
             mode: '0600',
-            content: %r{.*},
             notify: 'Exec[jboss-cli.sh --file=config.cli]',
             show_diff: 'false',
           )
         end
 
+        it do
+          is_expected.to contain_concat__fragment("config.cli-keycloak").with(
```
Comment (treydock) [inline spec/classes/init_spec.rb:133]: Use single quotes, rubocop is failing here. One way to fix if you have Ruby environment is `bundle exec rake rubocop:auto_correct`.

Comment (danifr) [reply spec/classes/init_spec.rb]: Great, thanks I didn't know it. I just `push -f` the fix.

**Pull request merged**